### PR TITLE
Make the landing page responsive via a layout-only CSS library

### PR DESCRIPTION
**Pull request by beekley**
## Description

Addresses: [Mobile Optimization](https://tables.area120.google.com/table/bX_kMNBW8LA35n6l5Hz_co/row/9ZLTEnkXfJeetgENepdkVs)

(well, the [library itself](https://bulma.io/documentation/overview/) is more than just layout, but it can be [imported modularly](https://bulma.io/documentation/overview/modular/) so we only use the layout stuff.)

Previously, we set a minimum width so the page would at least render legibly on mobile. However, this meant the UI was quite small and required side-scrolling on narrow displays. 

This PR adds Bulma, which is a CSS library I've used before, to make the UI responsive. I started writing CSS to do this, but it got complicated quickly. So I'd rather just import some CSS that others have written to do it in a more robust way than I could.

Future work:

* Match Copper's spacing/etc by [overriding Bulma's variables](https://bulma.io/documentation/customize/with-node-sass/).
* Make the other pages responsive.

### New

- Added a SASS library, Bulma.

### Changed

- Pretty much every landing page component uses Bulma for layout.

## Testing and Reviewing

You can play around with the UI on my sandbox: https://beekley.leadout-sandbox.blueconduit.com/

image

image

### CODE REVIEW

File: client/src/components/ResponsiveNav.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      [Titles.MAP_TITLE, router.MAP_ROUTE_BASE],
+      [Titles.ABOUT_TITLE, router.ABOUT_ROUTE],
+      [Titles.RESOURCES_TITLE, router.RESOURCES_ROUTE],
+    ];
+    return {
+      routes,
+    };
+  },
+  mounted() {
+    // Expand the menu via the burger button
```
Comment (kailamjeter) [inline client/src/components/ResponsiveNav.vue:43]: so this changes all navbars, not just mobile version?

File: client/src/components/landing_page/ExploreMapSection.vue [hunk 2]
```diff
@@ -1,26 +1,38 @@
 <template>
-  <div class='section'>
-    <div class='header-section'>
-      <div class='h1-header-large'>
-        {{ messages.EXPLORE_MAP_SECTION_HEADER }}
+  <div class='section is-small'>
+    <div class='columns is-centered has-text-centered'>
```
Comment (kailamjeter) [inline client/src/components/landing_page/ExploreMapSection.vue:3]: Not familiar with Bulma but it looks like these properties specify a lot of the properties we are currently specifying with CSS .. if this is the case can we remove duplicate css properties? (mixins for column, center, and text-align center all seem covered here)

File: client/src/components/ResponsiveNav.vue (hunk 1 above)
Comment (beekley) [reply client/src/components/ResponsiveNav.vue:43]: Short answer: yes

Longer answer: there is only one navbar. It just looks different in mobile vs desktop (desktop shows the links while mobile shows an expandable menu via a burger button). Bulma doesn't include any JS so we have to write our own to make the button work here.

File: client/src/components/landing_page/ExploreMapSection.vue (hunk 2 above)
Comment (beekley) [reply client/src/components/landing_page/ExploreMapSection.vue]: Yup! Done. Good idea.

**Review by kailamjeter — APPROVED** [review]

File: client/src/components/ResponsiveNav.vue (hunk 1 above)
Comment (kailamjeter) [reply client/src/components/ResponsiveNav.vue:43]: SG! Just wanted to make sure links still showed on desktop.

**Pull request merged**